blog-posts/gatsby-ssr-caching.md

```diff
@@ -0,0 +1,95 @@
+---
+path: "/blog/cache-gatsby-ssr-page-on-gatsby-cloud/"
+date: "2022-02-11"
+title: "Cache Gatsby SSR pages on Gatsby Cloud"
+featuredImage: images/gatsby-ssr-caching.jpg
+author: "Fabian Dietenberger"
+authorSummary: "Senior Developer at Satellytes"
+teaserText: Learn how to cache Gatsby SSR page on Gatsby Cloud
+seoMetaText: Lean how to cache Gatsby SSR pages on Gatsby Cloud to boost page speed. Your users won't see a difference to SSG or DSG pages anymore.
+leadboxText: Want to learn more about creating blazing fast pages? Then join us to learn more!
+attribution:
+    creator: Shiro Hatori
+    source: https://unsplash.com/photos/WR-ifjFy4CI
+---
+
+Since Gatsby 4 you can do [server side rendering](https://www.gatsbyjs.com/docs/how-to/rendering-options/using-server-side-rendering/) 
```

Comment:
Personally I think the start into the text is quite sudden. This could benefit from more introduction text.
This doesn't need to be sophisticated, I think even it shouldn't be. 
Just something nice and flowing to get you started, e.g., 
```
Ever since it's introduction [Gatsby](https://gatsbyjs.com] has established itself as the standard solution for React-based [Jamstack](https://jamstack.org/what-is-jamstack/) architectures. 
But where this approach has proven to be quite powerful for smaller sites, the restriction to pre-build all pages using [Static Site Generation](...) has proven to be a bottleneck [for larger websites](https://www.gatsbyjs.com/docs/how-to/performance/improving-build-performance/) or pages with real-time requirements. 

Luckily the recent v4 update has added a couple of interesting new tools to the developer toolbox. Time to give one of them - [server side rendering](https://www.gatsbyjs.com/docs/how-to/rendering-options/using-server-side-rendering/) - an in-depth-look and explore how to use it and what is can be done to optimize it for real live use-cases.

In contrast to the SSG-approach, where all content is pre-rendered at build-time, with [Server side rendering](https://www.gatsbyjs.com/docs/how-to/rendering-options/using-server-side-rendering/) content is rendered server whenever the user requests a page.